### DISCUSSION TITLE
Is there a replacement of usage accounts.at(address, force=True) in Ape

### DISCUSSION THREAD

**Discussion by wallenZhao2046** [ApeWorX/ape#2158] [Q&A] [answered]
I am Brownie user, and want to migrate to ape framework. There is an usage in brownie, "In a development environment, it is possible to send transactions from an address without having that addresses private key". To create an Account object from an arbitrary address, use the Accounts.at method and include force=True as a keyword argument".  

This is useful to simulate function in local fork mainnet network. My question is whether a replacement of this usage in ape framework?

**Comment by fubuloubu** [comment, marked as answer]
You can use this:

```py
address = accounts[<address>]
```

In local test networks and with forking, `address` will be an impersonated account, if impersonation is supported

**Reply by wallenZhao2046** [reply]
Thanks a lot! It works
